esp/src/eclwatch/stub.js

```diff
@@ -173,6 +178,8 @@ define([
                     });
                     idleWatcher.start();
                     monitorLockClick.unlocked();
+                } else if (dojo.cookie("ECLWatchUser")) { //this is due to the hard cache done by the browser
+                    window.location.replace(dojoConfig.urlInfo.basePath + "/Login.html");
                 }
                 stopLoading();
```

Comment:
Should  the ECLWatchUser cookie be cleared at this point?

Reply:
_onLogout() perhaps

Reply:
Should be cleared onLogout() I would assume.